### PR TITLE
feat: Host UI on ECS with ALB

### PR DESCRIPTION
**Pull request by kpark277**
Issue #, if available:

Description of changes:
* Setup ECS Cluster to host SWB UI with ALB
* Added script for building ECS container image with Docker
* Added Dockerfile to repo parent directory
* Updated proxy Lambda to handle backend requests made by ECS through ALB

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by SanketD92 — DISMISSED** [review]

File: solutions/swb-ui/ui/scripts/image-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (insignias) [inline solutions/swb-ui/ui/scripts/image-deploy.sh:1]: nit: #!/usr/bin/env bash

https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash

**Review by insignias — APPROVED** [review]

File: solutions/swb-ui/ui/scripts/image-deploy.sh (hunk 1 above)
Comment (kpark277) [reply solutions/swb-ui/ui/scripts/image-deploy.sh]: Updated.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM node:16-alpine
```
[the remark this answers is not in the document]
Comment (kpark277) [reply Dockerfile]: Moved to solutions/swb-ui/ui.

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.github
```
Comment (insignias) [inline .dockerignore:1]: Do we still need the `.dockerignore` file at the here ?

**Review by insignias — APPROVED** [review]

**Review by SanketD92 — APPROVED** [review]

**Pull request merged**